application.go

```diff
@@ -272,11 +351,45 @@ func (r *Application) CheckTCP(port, interval int) (*Application, error) {
 	health.IntervalSeconds = interval
 	health.PortIndex = portIndex
 	// step: add to the checks
-	r.HealthChecks = append(r.HealthChecks, health)
+	r.AddHealthCheck(*health)
 
 	return r, nil
 }
 
+// AddUris adds one or more uris to the applications
+//		arguments:	the uri(s) you are adding
+func (r *Application) AddUris(newUris ...string) *Application {
```

Comment:
you guessed right: I did this for consistency with the rest of this code. I would rather either do a linting PR or lint everything in this PR to minimize breaking changes PRs -- what do you think?